packages/terra-compact-interactive-list/src/subcomponents/Row.jsx

```diff
@@ -153,6 +159,7 @@ const Row = (props) => {
           columnMinimumWidth={columnMinimumWidth}
           columnMaximumWidth={columnMaximumWidth}
           widthUnit={widthUnit}
+          onCellSelect={() => onCellSelect(index)}
```

Comment:
This will cause an unnecessary rerender every time.  You should create a method and wrap with the useCallback hook to prevent this.

Reply:
Fixed in https://github.com/cerner/terra-framework/pull/1919/commits/fc32888e860735982a59eadfd00346f3b976688d